translations/translation_hu.xml

```diff
@@ -365,50 +365,50 @@ További nagy funkció a bálák összegyűjtése és becsomagolása a mezőn.
+Ha kiválasztottál egy CP munkát, a mező pozíciót a kiválasztott mezőre kell helyezned, hogy útvonalat generálj, vagy bálakeresőt használj rajta.
+A pozíció nagyjából meghatározza az útvonal kezdőpontját is.
+Ha a GIANTS segítőt szeretnéd arra használni, hogy a mezőhöz vezessen, akkor egy célpozíciót is be kell állítanod közel az útvonal kezdőpontjához.
+A sáveltolás opciót csak akkor használd, ha több segítőt akarsz dolgoztatni ugyanazon a mezőn. Ehhez kérlek nézd meg az alábbi különálló segítő oldalt.
 "/>
     <text name="CP_help_page_readyJobMenuFunctions_text" text="
-When the field position is placed correctly on a field, you will see the field border drawn on the map.
-If you are creating a CP: Fieldwork job, you will gain access to the course generator settings. 
+Ha a mező pozíció helyesen lett beállítva, a mező határai kirajzolódnak a térképen.
+Ha egy  CP: mezőmunkát hozol létre, hozzáférést nyersz az útvonal generátor beállításaihoz. 
 "/>
     <text name="CP_help_page_JobMenuExtras_text" text="
-Now it's possible to start the driver directly from the menu. The Giants helper will drive to the target position and from there the CP job automatically takes over.
-Alternatively you can also start the driver from the HUD if you are close to the field with the vehicle or using the AutoDrive mod to deliver the CP job close to the field.
+Mostmár lehetséges a sofőr indítása közvetlenül a menüből. A GIANTS segítő a célpozícióra fog vezetni és onnan a CP munka automatikusan átveszi az irányítást.
+Alternatív megoldásként a sofőrt a kijelzőről is indíthatod, ha a járművel közel vagy a mezőhöz, vagy AutoDrive modot használsz, hogy a CP munkát közel vidd a mezőhöz.
 "/>
-    <text name="CP_help_page_courseGenerator_title" text="Course generator"/>
+    <text name="CP_help_page_courseGenerator_title" text="Útvonal generátor"/>
     <text name="CP_help_page_courseGeneratorBase_text" text="
-Every fieldwork course is generated by the course generator. This is a mighty tool, which allows you to customize the generated course to your needs.
-If you are new to this, you should probably start by only changing the headland amount setting.
-After that you can always change the other values and hit generate course and have a look, what impact certain values have on the course.
-Experimenting with it and check what impact the different setting on different fields have is the best way to get used to it.
+Minden mezőmunka útvonalat az útvonal generátor hoz létre. Ez egy remek eszköz, ami megengedi a generált útvonalak igény szerinti testreszabását.
+Ha új vagy ebben, valószínűleg azzal kellene kezdened, hogy csak a fordulósávok számát módosítod.
+Ezt követően mindig változtathatsz más értékeket és generálhatsz útvonalat, hogy megnézd, egyes értékek milyen hatással vannak az útvonalra.
+A kísérletezés és különböző beállítások hatásainak megfigyelése különböző mezőkön a legjobb módja, hogy hozzá szokj ehhez.
```

Comment:
hozzászokj

Reply:
Rendben, ha ezeket jobbnak látod, felőlem mehet. Valahol a leírásban szakértő módot írtam, több helyen. Ahelyett is mehet haladó mód. Egy korábbi fordításban valaki azt írta.

Reply:
Én sem vagyok szakmabeli, ezért nem ismerem a szakzsargont.

Reply:
Köszönöm, akkor tolsz fel egy módosított változatot?

Reply:
Olyat még nem csináltam. Mármint egy benyújtott fordítás módosítását. Kell egy kis idő, amíg megtalálom, hogyan lehet változtatni.

Reply:
Vagy az egészet újra be kell küldeni?

Reply:
Lehet módosítani, persze, csak ugyanoda kell még egy push a változtatásokkal, de így is jó.